core/MyTransport.cpp

```diff
@@ -553,6 +554,7 @@ bool transportRouteMessage(MyMessage &message)
 			}
 			TRANSPORT_DEBUG(PSTR("!TSF:RTE:N2N FAIL\n"));
 		}
+#endif
```

Comment:
How about adding an else clause here, with something like
`TRANSPORT_DEBUG(PSTR("!TSF:RTE:N2N DIS\n"));` so people who enable this feature and forget about it (or don’t fully realize what it does) can got some information on why their N2N messages disappear?

Reply:
Okay, I will try it out and check how it works.